app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/HomePresenter.kt

```diff
@@ -37,9 +38,9 @@ internal class HomePresenter(
     var isReloading by remember { mutableStateOf(lastState.isReloading) }
     var successData: SuccessData? by remember { mutableStateOf(SuccessData.fromLastState(lastState)) }
     var loadIteration by remember { mutableIntStateOf(0) }
-    var importantMessagesIteration by remember {
-      mutableIntStateOf(0)
-    }
+    val alreadySeenImportantMessages: List<String>
+      by seenImportantMessagesStorage.seenMessages.collectAsState(initial = emptyList())
```

Comment:
And then here you would even be able to avoid the `initial = emptyList()` part here if it's a StateFlow